### PR TITLE
Add a few tooltips

### PR DESCRIPTION
**Pull request by lkiesow**
Starting to use audiobookshelf, the function of some buttons weren't very clear to me and while some buttons have tooltips, others have not.

This patch adds some additional tooltips to the user interface, further explaining some of the functionality.

_This doesn't modify any of the translations since that's often a separate process. I can adjust the other files if necessary._

### CODE REVIEW

File: client/strings/en-us.json [hunk 1]
```diff
@@ -282,6 +284,7 @@
   "LabelRecentSeries": "Recent Series",
   "LabelRegion": "Region",
   "LabelReleaseDate": "Release Date",
+  "LabelRemoveCover": "Remove book cover",
```
Comment (advplyr) [inline client/strings/en-us.json:287]:
```suggestion
  "LabelRemoveCover": "Remove cover",
```
This is also used for podcasts

File: client/strings/en-us.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
   "ButtonOk": "Ok",
   "ButtonOpenFeed": "Open Feed",
   "ButtonOpenManager": "Open Manager",
+  "ButtonPause": "Pause",
```
Comment (advplyr) [inline client/strings/en-us.json:42]:
```suggestion
```
not used

**Comment by advplyr** [comment]
Thanks!

**Comment by lkiesow** [comment]
Thanks for the quick review and sorry for missing `ButtonPause`

**Pull request merged**